### ISSUE TITLE
External source load

### ISSUE THREAD

**Issue by ocdogan** [gitana/alpaca#599]
External sources defined as absolute path directing the existing path with the loaded page does not work. Atleast **"./"** file path prefix needs to be given to make it work as an external URL.

Not working style:
----------------------------------------------------------------------
$("#form").alpaca({
		"schemaSource": **"connector-custom-schema.json"**,
		"optionsSource": **"connector-custom-options.json"**,
		"dataSource": **"connector-custom-data.json"**, 
		"viewSource": **"connector-custom-view.json"**
	});

Working style:
----------------------------------------------------------------------
$("#form").alpaca({
		"schemaSource": **"./connector-custom-schema.json"**,
		"optionsSource": **"./connector-custom-options.json"**,
		"dataSource": **"./connector-custom-data.json"**, 
		"viewSource": **"./connector-custom-view.json"**
	});

**Comment by seanvree** [comment]
Yup, I found this too.  I'm assuming a "base URL" would need be defined?  But it is weird that it doesn't use the relative URL when defined.
